package.json

```diff
@@ -140,6 +141,16 @@
         "NO_ESLINT": "1"
       }
     },
+    "test-ci-next": {
```

Comment:
Maybe leave a comment for posterity

Reply:
json doesn't allow comments 😞, but git blame should help.